packages/babel-plugin-transform-typescript/test/fixtures/optimize-const-enums/export-const-enum-type-no-deopt/stderr.txt

```diff
@@ -0,0 +1,16 @@
+The exported identifier "WhitespaceFlag" is not declared in Babel's scope tracker
```

Comment:
The test case is
```ts
const enum WhitespaceFlag {
  before = 1 << 0,
  after = 1 << 1,
}

export type { WhitespaceFlag as WF1 };
export { type WhitespaceFlag as WF2 };

export const before = WhitespaceFlag.before;
```

We first remove the enum declaration `WhitespaceFlag`, so the referenced `WhitespaceFlag` becomes invalid and the warning is triggered. In a way it is similar to #14818 where we removed the scope binding info but didn't fix its now-dangling referenced paths.